### ISSUE TITLE
dragging=false + tap=false keep touch-action:none in map div

### ISSUE THREAD

**Issue by scherebedov** [Leaflet/Leaflet#4415]
Hi. I have some problems with scroll page with leaflet map in mobile using map dom element.
If I do init with parameters `dragging: false` and `tap: false`, I can't scroll page using map dom element, beacause in file `leaflet.css` 

```
.leaflet-container {
    overflow: hidden;
    -ms-touch-action: none;
    touch-action: none;
    }
```

Maybe set this css property to auto, if dragging and tap is false ? Thank's.

**Comment by IvanSanchez** [comment]
Not sure about this. Even when the dragging handler and tap handler are disabled, some other handlers (e.g. scrollwheel or touchzoom) might still capture the default events.

Using `preventDefault` on any events might work, but I think there would be far too many edge cases.

In the meantime, I suggest you override those CSS rules on your map div.

**Comment by lazopm** [comment]
Running into this issue, would anything break if I remove that rule?

**Comment by IvanSanchez** [comment]
@lazopm Yes. The webpage would scroll when a user would try to zoom using the mouse wheel.

**Comment by lazopm** [comment]
@IvanSanchez I'm talking about `.leaflet-container { touch-event: none }` zoom works fine for touch and scroll without it on Chrome.
Looks like the rule was recently added to fix a Spartan zoom issue e44179d
In my opinion it's probably best to find a different solution for the Spartan bug and drop the css rule.

Thoughts?

**Comment by lazopm** [comment]
Actually the commit is a year old, I'm gonna check it's still an issue on Edge.

**Comment by lazopm** [comment]
Zoom seems to be working fine on Edge and Windows Phone without touch-action: none.
I created a PR to remove the line.
#4472

**Comment by jfirebaugh** [comment]
This was fixed by #4552.
